src/test/java/edu/harvard/iq/dataverse/api/DataverseFeaturedItemsIT.java

```diff
@@ -0,0 +1,112 @@
+package edu.harvard.iq.dataverse.api;
+
```

Comment:
Heads up that some tests are failing at https://jenkins.dataverse.org/job/IQSS-Dataverse-Develop-PR/job/PR-11124/36/testReport/

Reply:
I don't have access to Jenkins. Can you tell me exactly which tests are failing? I can also request temporal access to Jenkins.

Reply:
As I mentioned in the frontend meeting, stuff like DataverseFeaturedItemsIT.testUpdateFeaturedItem is failing. 😢

Reply:
Fixed. Thanks for noticing!

Reply:
Closer! But still some failures at https://jenkins.dataverse.org/job/IQSS-Dataverse-Develop-PR/job/PR-11124/40/testReport/

![Screenshot 2025-01-24 at 1 48 43 PM](https://github.com/user-attachments/assets/80065274-e01c-4039-87ca-dbe16ae07d72)

Reply:
It should be fixed now. I have simplified the assertion.

Reply:
Tests are passing! Thanks!